lib/build/tasks.js

```diff
@@ -129,7 +139,8 @@ function watchAssets (sourcePath, targetPath) {
 }
 
 function watch () {
-  watchSass(appSassPath, libSassPath, appCssPath)
+  watchSass(appSassPath, libSassPath, appCssPath, libSassOptions)
```

Comment:
do we need to watch the sass in lib? Since users shouldn't be changing it

Reply:
This is true, but we change when working on the kit.   What do you think @natcarey and @lfdebrux?

Reply:
any thoughts @natcarey ? For me its less about performance and more about maintaining code we don't need

Reply:
We could "detect" that this is a development branch and only watch the lib files if that applies?

Reply:
I'm going to say let's leave it for now, we've got a new world coming where `lib` won't be in their project so let's review this then.